spec/defines/resource_server_spec.rb

```diff
@@ -904,18 +904,6 @@
         it { is_expected.to contain_nginx__resource__location("#{title}-default").with_ssl_only(true) }
       end
 
-      context 'SSL cert missing' do
```

Comment:
that should be handled by the reviewer + the existing tests. They will fail when wrong datatypes are supplied.